### PR TITLE
feat: replace Get-Item powershell cmd with golang api on Windows HostProcess mode

### PR DESCRIPTION
**Pull request by umagnus**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

feat: replace Get-Item powershell cmd with golang api on Windows HostProcess mode

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/mounter/safe_mounter_host_process_windows.go [hunk 1]
```diff
@@ -142,7 +142,6 @@ func (mounter *winMounter) Unmount(target string) error {
 		klog.V(2).Infof("remote server path: %s, local path: %s", remoteServer, target)
 		if hasDupSMBMount, err := smb.CheckForDuplicateSMBMounts(driverGlobalMountPath, target, remoteServer, mounter.volStatsCache); err == nil {
 			if !hasDupSMBMount {
-				remoteServer = strings.Replace(remoteServer, "UNC\\", "\\\\", 1)
```
Comment (andyzhangx) [inline pkg/mounter/safe_mounter_host_process_windows.go:145]: why remove this line? could you also paste the different behavior using powershell command and `os.Readlink` on a smb mount?

Comment (umagnus) [reply pkg/mounter/safe_mounter_host_process_windows.go:145]: powershell command is `c:\var\lib\kubelet\plugins\kubernetes.io\csi\file.csi.azure.com\cf2798d1325a0734ebe0a4c97cbee6ad5b8c1874e9b9c6d3d373d4e8dd016b81\globalmount -> UNC\f005e0c98fa2041c09d6df3.file.core.windows.net\pvc-572ab4d0-bc95-4df3-8de6-749726d87aa3`
and `os.Readlink` is `c:\var\lib\kubelet\plugins\kubernetes.io\csi\file.csi.azure.com\cf2798d1325a0734ebe0a4c97cbee6ad5b8c1874e9b9c6d3d373d4e8dd016b81\globalmount -> \\f005e0c98fa2041c09d6df3.file.core.windows.net\pvc-572ab4d0-bc95-4df3-8de6-749726d87aa3`

Comment (andyzhangx) [reply pkg/mounter/safe_mounter_host_process_windows.go:145]: pls squash commits, thx

**Pull request merged**